Multisig/Presentation/UI Library/QRScanner/QRCodeScanner.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  QRCodeScanner.swift
+//  Multisig
+//
+//  Created by Moaaz on 4/15/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct QRCodeScanner: UIViewControllerRepresentable {
+    var header: String?
+    var handler: (String) -> Void
+    func makeCoordinator() -> Coordinator {
+        Coordinator(self)
+    }
+    
+    func makeUIViewController(context: Context) -> QRCodeScannerViewController {
+        let scannerViewController = QRCodeScannerViewController(nibName: "QRCodeScannerViewController", bundle: nil)
+        scannerViewController.delegate = context.coordinator
+        scannerViewController.header = header
+        scannerViewController.setup()
+        
+        return scannerViewController
+    }
+    
+    func updateUIViewController(_ vc: QRCodeScannerViewController, context: Context) {
+    }
```

Comment:
please add a comment to the empty body
```suggestion
        // do nothing
    }
```